django/db/__init__.py

```diff
@@ -36,6 +37,50 @@
 ]
 
 connections = ConnectionHandler()
+async_connections = AsyncConnectionHandler()
+
+
+class new_connection:
```

Comment:
Are we OK with `new_connection` as a name? Seems odd, _but_ also an eminently fixable problem downstream

Reply:
Sounds good to me as justification! Please ignore this comment then 👍